packages/dds/tree/src/tableSchema.ts

```diff
@@ -71,6 +73,23 @@ export interface RowPrivate<
 	readonly cells: TreeRecordNode<TCell>;
 }
 
+function allowedTypesListStringFromImplicitFieldSchema(
+	fieldSchema: ImplicitFieldSchema,
+): string {
+	if (fieldSchema instanceof FieldSchema) {
+		return allowedTypesListStringFromImplicitAllowedTypes(fieldSchema.allowedTypes);
```

Comment:
@CraigMacomber this doesn't capture the field schema aspects. Do we have any other examples where we would want to include such information?